src/app/screens/services/Timetable.tsx

```
@@ -0,0 +1,351 @@
+  function getNowTimeForLine() {
+    const now = new Date();
+    const currentHour = now.getHours();
+    const currentMinutes = now.getMinutes();
+    const minutesSinceStart = (currentHour - START_HOUR) * 60 + currentMinutes;
+    return (minutesSinceStart / 60) * HOUR_HEIGHT + 10; // +10 pour que la ligne soit alignés avec les horaires
+  }
+
+  const [nowTimeLine, setNowTimeForLine] = useState(getNowTimeForLine());
+
+  useEffect(() => {
+    const interval = setInterval(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: getNowTimeForLine

This dependency is not specified in the hook dependency list.

Either include it or remove the dependency array

(lint/correctness/useExhaustiveDependencies)